src/java/org/apache/cassandra/utils/ObjectSizes.java

```diff
@@ -39,6 +42,8 @@
 
     private static final long DIRECT_BUFFER_HEAP_SIZE = measure(ByteBuffer.allocateDirect(0));
 
+    public static final long IPV6_SIZE = ObjectSizes.measureDeep(new InetSocketAddress(getIpvAddress(16), 42));
```

Comment:
should be something like `IPV6_SOCKET_ADDRESS_SIZE`, your name doesn't include the fact this includes port